src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/PersonTransformer.kt

```diff
@@ -68,6 +69,24 @@ class PersonTransformer {
     )
   }
 
+  fun transformProbationOffenderToPersonApi(probationOffenderResult: ProbationOffenderSearchResult.Success.Full, nomsNumber: String) =
+          FullPerson(
+            type = PersonType.fullPerson,
+            crn = probationOffenderResult.probationOffenderDetail.otherIds.crn,
+            name = "${probationOffenderResult.probationOffenderDetail.firstName} ${probationOffenderResult.probationOffenderDetail.surname}",
+            // TODO: we are not handling if DoB is not there, what should we do?
+            dateOfBirth = probationOffenderResult.probationOffenderDetail.dateOfBirth!!,
```

Comment:
Having that `!!` there makes me nervous. Could you extend the CAS2 `FullPerson` to make `dateOfBirth` optional?

Reply:
I'm using the shared `FullPerson` model at the moment, I'm wondering if I should create a CAS2 version or if there's a way to extend the `FullPerson` with just the change to that one field

Reply:
after chatting with @edavey we decided to take the risk of having the !! for now as it's very unlikely we won't have DoB (we're assuming!), and we don't want to create yet another model for this one difference to `FullPerson`,  and I'm also thinking it should be required as a piece of data that identifies the right person.